controls/radvirtualgrid/mvvm.md

```diff
@@ -0,0 +1,327 @@
+---
+title: MVVM Support
+page_title: MVVM Support
+description: MVVM Support
+slug: virtualgrid-mvvm
+tags: virtualgrid,mvvm
+published: True
+position: 1
+---
+
+# MVVM Support
+
+This topic will go through the approach of using __RadVirtualGrid__ in an MVVM environment. This can be achieved by benefiting from the __DataProvider__ mechanism of the control. More information on this matter can be found in the [Getting Started]({%slug virtualgrid-getting-started2%}) and [Custom DataProvider]({%slug virtualgrid-custom-dataprovider%}) topics.
```

Comment:
I am not sure if MVVM can be considered as an environment. I would use 'scenario' instead.